### PR TITLE
[readme.js] Add more tests for duplicate tags

### PR DESCRIPTION
**Pull request by mikeharder**
- Tests related to #37003 (which we decided to not fix)

### CODE REVIEW

File: .github/shared/test/readme.test.js [hunk 1]
```diff
@@ -26,8 +26,6 @@ describe("readme", () => {
     expect(readme.path).toBe(resolve("/specs/foo/readme.md"));
   });
 
-  // TODO: Test that path is resolved against backpointer
```
Comment (mikeharder) [inline .github/shared/test/readme.test.js:29]: test already added, forgot to remove comment
